app/views/profile/_pii.html.slim

```diff
@@ -7,10 +7,12 @@
     .clearfix.mxn1
       .sm-col.sm-col-5.px1 = t('profile.index.address')
       .sm-col.sm-col-7.px1
-        | #{decrypted_pii.address1}<br>
-        - if decrypted_pii.address2.present?
-          | #{decrypted_pii.address2}<br>
-        | #{decrypted_pii.city}, #{decrypted_pii.state} #{decrypted_pii.zipcode}
+        = render 'shared/address', { \
+            address1: decrypted_pii[:address1],
```

Comment:
Yeah I guess so? I was imagining this was more flexible since could accommodate either an object or a hash, but since we both objects are hash-like for now, a hash makes sense. Will fix!